packages/dag4-network/src/dto/v2/snapshot.ts

```diff
@@ -66,3 +68,18 @@ export type SnapshotL0 = {
   value: SnapshotL0Value;
   proofs: Proof[];
 };
+
+export type CurrencySnapshotV2 = {
+  hash: string;
+  ordinal: number;
+  height: number;
+  subHeight: number;
+  lastSnapshotHash: string;
+  blocks: string[];
+  epochProgress: number;
+  timestamp: string;
+  fee: number;
```

Comment:
This value is not present on all ML0 snapshots, please type as nullish => value?: {type} | null.